soh/soh/SohMenuBar.cpp

```diff
@@ -155,6 +155,13 @@ void DrawShipMenu() {
         }
 #if !defined(__SWITCH__) && !defined(__WIIU__)
         UIWidgets::Spacer(0);
+#if defined(__APPLE__) || defined(__WINDOWS__) || defined(__LINUX__)
```

Comment:
The `#if`s here feel funky. Seems like this "open folder" button should be placed above the `#if` for "Quit". Could also just collapse it together in one `#if`.

Reply:
Good point. I was thinking for the future if there was something that was added that wasn't the PC platforms. Just forgot that wasn't very likely XD.